docs/snap/instructions.md

```diff
@@ -135,6 +122,9 @@ If you want a service user to be able to purchase crypto inside the widget using
 
 Once whitelisted you will need to use [this extension](https://chromewebstore.google.com/detail/always-disable-content-se/ffelghdomoehpceihalcnbmnodohkibj) for local testing, or ensure that your dapp is locally accessible via `http://localhost` or `https://localhost` (i.e. no port number in the URL).
 
+## Icon
+As an icon for Metamask provider in your dapp, you can download this [svg](../images/cardano_metamask.svg).
```

Comment:
This link does not work for me. 

Anyway we need to keep the content inside the `snap` folder to be properly copied to gitbook.

Reply:
Sorry, forgot to update this one.